src/maps/Doc4Metrics.py

```diff
@@ -0,0 +1,28 @@
+{
+"IO":{
+    "WriteTotalMB": "Total storage layer written data in MB.",
+    "readPercentageOps" : "% operations by the ROOT layer vs storage.",  
+    "readAverageKB": "Read rate of storage layer in KB/operation.",
+    "readTotalMB": "Total storage layer read in MB.", 
+    "readNumOps": "Storage-based read operation",
+    "readCachePercentageOps": "% successful operations by ROOT layer vs storage.", 
+    "readMBSec": "Read rate MB/millisecond.",
+    "readMaxMSec": "The longest read time in millisecond.",
+    "readTotalSecs": "Total time for root-based read in seconds.",
+    "writeTotalSecs": "Total writting time in seconds."
+            },
+"CPU":{   
+    "MinEventTime":"Shortest event wall time in seconds.",
+    "MaxEventTime":"Longest event wall time in seconds",
+    "AvgEventTime":"Average event wall time in seconds.",
+    "EventThroughput":"Num of events per second.",
+    "TotalJobTime":"Job wall time in seconds.",
+    "TotalJobCPU":"Job CPU in seconds.",
+    "TotalLoopCPU":"in seconds."
+},
+"memory":{
```

Comment:
memory is not part of storage. It is a separated part parallel to storage.  The schema I use is https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/Services/WMArchive/DataMap.py#L39. If not,  please show me which schema.